### ISSUE TITLE
What is the purpose of  `try ... catch` for all `require` statements in `src/*.js`?

### ISSUE THREAD

**Issue by cuixiping** [LingDong-/wenyan-lang#409]
## Describe your issue/suggestions

I noticed that there are `try ... catch` for all `require` statements in `src/*.js`.
What is the purpose of doing this?
@LingDong-  Thank you.

Fox example

```
try {
  var fs = require("fs");
  var { semantic } = require("./highlight");
  var { num2hanzi } = require("./hanzi2num");
  var parser = require("./parser");
} catch (e) {}
```

## Checklist

Before you create this issue, Please make sure ALL the following items are checked

- [x] If this is a feature request, it's NOT listed [Here](https://github.com/LingDong-/wenyan-lang#feature-requests)
- [x] If this is a bug report, it's NOT listed [Here](https://github.com/LingDong-/wenyan-lang#known-bugs)

**Comment by antfu** [comment]
For browser compatibility. Currently, `ide` and `site` are using code comes from simply concating all the files in src/*js.  You can check this out: 

https://github.com/LingDong-/wenyan-lang/blob/1cecae9de1919486f34241f379248ed402b4fa96/tools/utils.js#L12-L33

**Comment by cuixiping** [comment]
May be we can use webpack or rollup to convert those files for web.
This writing looks strange and can bring unknown edge effects.

**Comment by antfu** [comment]
Yes we will, eventually, I suppose. As we are already using webpack to build the npm packages. However as for now, current workflow make @LingDong- easier to work and focus on the language itself.
